packages/salesforcedx-vscode-core/package.json

```diff
@@ -406,6 +407,11 @@
           "type": ["boolean"],
           "default": true,
           "description": "%show_cli_success_msg_description%"
+        },
+        "salesforcedx-vscode-core.isv-debugger.enabled": {
```

Comment:
Is this hidden when you go to VS Code > Preferences? Or is this visible to everyone? There is a way to hide the preferences that @GaryFNC used last time.

This is not a big deal but if you expose it, you need to be prepared for the fact that the user might have this setting in the preference file, which could be stale when we remove it in the future.

Reply:
@vazexqi @DatGuyJonathan The setting is currently exposed. It's grouped under "Salesforce Feature Previews". I think this was supposed for this kind of stuff. However, it should be possible to simply remove the declaration from `package.json` and that it will not be exposed. We can do that in a subsequent review.

![2018-02-05_11-07-55](https://user-images.githubusercontent.com/191472/35799129-1334de26-0a65-11e8-8e41-e1a8718a0bff.png)

Reply:
>I think this was supposed for this kind of stuff.

Hmmm, I'm not quite sure I follow. What do you mean? 

But, either way, I think you are right that if you just remove the declaration from package.json, it will not be exposed (@GaryFNC). You will still access the preference the same way just that it won't be exposed. 

*The question now is that since we are cutting the branch at 6 pm PT, are you going to create a new PR now?*

One of the reasons why I do not really want the preference known at this particular point is that the feature doesn't do much yet. So exposing it as a preference might cause more confusion.

The contributions not only appear in the settings but it also appears on the marketplace. See attached screenshot for the Apex editor case.

<img width="1552" alt="screen shot 2018-02-05 at 7 37 56 am" src="https://user-images.githubusercontent.com/54414/35813118-a2133b74-0a47-11e8-885b-dab39ab7abba.png">

The %...% is due to https://github.com/Microsoft/vscode/issues/31993 which has not been fixed yet – we might want to fallback on not localizing the preference since it is quite confusing ot see the %..%

Reply:
@vazexqi Thanks for bringing the marketplace issue up. I'll create a PR ASAP.